tests/rafter/pkg/file-client/file-client.go

```diff
@@ -0,0 +1,104 @@
+package fileclient
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type Config struct {
+	Address   string `envconfig:"default=storage.kyma.local"`
+	Secure    bool   `envconfig:"default=true"`
```

Comment:
Needed? Use address and protocol from Asset status.

Reply:
Done :)